CONTRIBUTING.md

```diff
@@ -58,6 +66,12 @@ nango init
 ```
 
 Change the .env file `NANGO_SECRET_KEY_DEV` and `NANGO_HOSTPORT`.
+
+```sh
+NANGO_SECRET_KEY_DEV="secret key from nango dashboard"
+NANGO_HOSTPORT="host:port"   # can be https://nango.dev or if running locally http://.localhost:3000
+```
```

Comment:
I'm not sure it's relevant, it's already what the .env is stating by default

Reply:
I did not find NANGO_HOSTPORT anywhere in .env, thats why i included it. the `NANGO_SECRET_KEY_DEV` however can be skipped.

Reply:
the `.env` mentioned here is in `nango-integrations`, when you launch `nango init` it should have created a .env file with all the mentioned var. cf:
https://github.com/NangoHQ/nango/blob/644fc71035dd8d86cd7f9c31fb1eb32649e41801/packages/cli/lib/cli.ts#L239-L250

Reply:
you are correct. Mistake on my part. I did not go deep enough. Sorry for the trouble.

Reply:
no worries ☺️